packages/arb-token-bridge-ui/src/util/withdrawals/fetchWithdrawals.ts

```diff
@@ -44,10 +44,9 @@ export const fetchWithdrawals = async ({
   fromBlock,
   toBlock
 }: FetchWithdrawalsParams) => {
-  const address = sender ?? receiver
-
-  if (typeof address === 'undefined') return []
-  if (!l1Provider || !l2Provider) return []
+  if (typeof sender === 'undefined' && typeof receiver === 'undefined') {
```

Comment:
NIT: We have removed the `if (!l1Provider || !l2Provider) return []` check. 
Although we won't run into that condition, IMHO it's good to have it as a fail-safe just in case.

Reply:
I've removed that because the types said they can't be null or undefined, which means we had bad typing

Reply:
I don't think providers should be undefined here, and if they are I'd just let subgraphs fail because there's a problem elsewhere.

Reply:
Makes sense, TS handles that for us.